### ISSUE TITLE
[BUG] Curious about gradient_acc

### ISSUE THREAD

**Issue by SungjunKim1** [deepinsight/insightface#2258]
Deviding by gradient_acc is missing when gradient_acc applied. Am I right?

**Comment by anxiangsir** [comment]
Yes, you are right. I will fix the bug in the code later. Interestingly, even though I used this buggy code, I was still able to train a good model:

https://raw.githubusercontent.com/anxiangsir/insightface_arcface_log/master/pfc03_wf42m_vit_b_8gpu/training.log

![image](https://user-images.githubusercontent.com/31175974/223721146-e7a4cfa0-f734-4cf4-b6ae-c5500e884230.png)

**Comment by SungjunKim1** [comment]
Also, total_batch_size should be cfg.batch_size * world_size * gradient_acc. Is right?
